core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -255,6 +255,10 @@ private SqlLibraryOperators() {
           OperandTypes.STRING)
           .withFunctionType(SqlFunctionCategory.STRING);
 
+  /** The "STRPOS(value1, value2)" function. */
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction STRPOS = new SqlPositionFunction("STRPOS");
```

Comment:
It looks like same function is available in Postgres[1]

[1] https://www.postgresql.org/docs/9.1/functions-string.html

Reply:
Oh yes, sorry about that. Should be fixed now.